.travis.yml

```diff
@@ -29,5 +29,7 @@ install:
   - sudo make -C ./buildtree install
 
 script:
+  - export CGO_CPPFLAGS="-I${GOPATH}/src/github.com/singularityware/singularity/builddir -I${GOPATH}/src/github.com/singularityware/singularity/src/runtime/c -I${GOPATH}/src/github.com/singularityware/singularity/src/runtime/c/lib"
```

Comment:
My expertise incoming:
`singularity/builddir` -> `singularity/buildtree`

Reply:
Ah, OK, as discussed, changing Travis to use the default option to `mconfig` since it's less confusing!